src/platform-includes/distributed-tracing/how-to-use/dotnet.mdx

```diff
@@ -0,0 +1,13 @@
+If you use the current version of our .NET SDK, distributed tracing works automatically with the following frameworks:
+
+- ASP.NET Core
+- Azure Functions Worker
+
+When using ASP.NET you will need to update your `Application_BeginRequest` method:
+
+```csharp
+protected void Application_BeginRequest()
+{
+    Context.StartOrContinueTrace();
```

Comment:
We [document](https://docs.sentry.io/platforms/dotnet/guides/aspnet/performance/included-instrumentation/) `StartSentryTransaction` already.

How do these two play together? User must pick one? 
Could the behavior of `StartOrContinueTrace` be added to `StartSentryTransaction` so users don't need to change anything?

Reply:
Yes. That is exactly how it works. For users using performance, there are no changes required. This is specifically for tracing without performance.